import-service/serverless.ts

```diff
@@ -0,0 +1,77 @@
+import type { Serverless } from 'serverless/aws';
+
+import { BUCKET_ARN, BUCKET_NAME } from './constants';
+
+const serverlessConfiguration: Serverless = {
+  service: {
+    name: 'import-service',
+    // app and org for use with dashboard.serverless.com
+    // app: your-app-name,
+    // org: your-org-name,
+  },
+  frameworkVersion: '2',
+  custom: {
+    webpack: {
+      webpackConfig: './webpack.config.js',
+      includeModules: true
+    },
+  },
+  plugins: [
+    'serverless-webpack',
+    'serverless-offline',
+  ],
+  provider: {
+    name: 'aws',
+    runtime: 'nodejs12.x',
+    apiGateway: {
+      minimumCompressionSize: 1024,
+    },
+    environment: {
+      AWS_NODEJS_CONNECTION_REUSE_ENABLED: '1',
+    },
+    region: 'eu-west-1',
+    iamRoleStatements: [{
+      Effect: 'Allow',
+      Action: 's3:ListBucket',
+      Resource: BUCKET_ARN,
+    },{
+      Effect: 'Allow',
+      Action: 's3:*',
+      Resource: `${BUCKET_ARN}/*`,
+    },],
+  },
+  functions: {
+    importProductsFile: {
+      handler: 'handler.importProductsFile',
+      events: [
+        {
+          http: {
+            method: 'get',
+            path: 'import',
+            cors: {
+              origins: '*',
+            },
```

Comment:
no query parameter described here

Reply:
sorry, for what purpose?

Reply:
Well, as said in our task "The name of CSV file with products will be passed in a query string as a name parameter and should be described in the serverless.yml file as a request parameter."